.github/workflows/verify-build.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Verify Build
+on:
+  push:
+    branches: ["main"]
```

Comment:
We'd want all branches right?

Reply:
I think we can simply remove the `branches` filter, and backport to all release branches, so that it runs on all branches.